Common/Field/include/Field/MagneticField.h

```diff
@@ -57,10 +57,8 @@ class MagneticField : public FairField
                   MagFieldParam::BMap_t maptype = MagFieldParam::k5kG,
 		  MagFieldParam::BeamType_t btype = MagFieldParam::kBeamTypepp,
 		  Double_t benergy = -1, Int_t integ = 2,
-                  Double_t fmax = 15, const std::string path = std::string(gSystem->ExpandPathName("$(O2_ROOT)"))
-		  + std::string("/share/Common/maps/mfchebKGI_sym.root")
-    );
-
+                  Double_t fmax = 15,
```

Comment:
would put a `;` here instead of `,`. Can you also still fix the clang-format problems as indicated by travis?

Reply:
@sawenzel : why? The string is an argument of the constructor, with its default value.
For the clang-format: not sure how to apply these particular corrections in travis report since it uses different clang version (and the suggestions look quite ridiculous), so simply applied 
````clang-format -style=file -i  Common/Field/include/Field/MagneticField.h Common/Field/src/MagneticField.cxx````